### PR TITLE
Event loop questions.

### PR DESCRIPTION
**Pull request by gthacoder**
Adds two questions related to the event loop and synchronous/asynchronous task execution in JavaScript.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Why would you use something like the `load` event? Does this event have disadvantages? Do you know any alternatives, and why would you use those?
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
+* Explain how synchronous and asynchronous functions are executed in JavaScript. How does event loop work? What is the difference between call stack and task queue?
```
Comment (darcyclarke) [inline README.md]: Can you break up these questions a bit?

Comment (arthurvr) [inline README.md]: ping @gthacoder

**Comment by arthurvr** [comment]
Would be nice if you could handle the merge conflict as well.

**Comment by arthurvr** [comment]
Hey @gthacoder still interested in picking this up?

**Comment by gthacoder** [comment]
@darcyclarke @arthurvr Sorry for my late reply. Sure, I will break up these questions and resolve the conflicts.

**Comment by richgilbank** [comment]
@gthacoder Any ETA on when you might be able to tackle this?

**Comment by gthacoder** [comment]
Resolved the merge conflicts and split the theoretical part in two.

**Comment by darcyclarke** [comment]
@gthacoder Still see a conflict. Can you try one more time?

**Comment by gthacoder** [comment]
@darcyclarke Done.

**Pull request merged**